docs/src/scss/_typography.scss

```diff
@@ -53,6 +53,10 @@ tbody {
   }
 }
 
+p a + a {
+  margin-left: 1rem;
```

Comment:
giving adjacent urls some space between them

Reply:
should we try using classnames?

Reply:
its in the md file, thats why didnt gave any classnames

Reply:
ok

Reply:
since this rules for markdown content only,  we should apply css property using `.markdown` classname

Reply:
same comments for other css rules which were applied with intend of applying on markdown content

Reply:
> same comments for other css rules which were applied with intend of applying on markdown content

does that mean for all the styles which come inside of `.markdown`?

Reply:
yes,

I know this will be big change. :) 
If you can please do this or do it in separate PR